### PR TITLE
Image and input components

### PR DESCRIPTION
**Pull request by parvez-rashid**
Created the image and input components. These are functional but will work on further styling and functionality after merge on dev. View screenshots below.
responsive-image

image

### CODE REVIEW

File: image/image.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    <div class="img-container">
+      <img
+        class="responsive-img"
+        src="https://picsum.photos/700/350"
+        alt="demo-image"
+      />
+    </div>
+    <h3>Round image</h3>
+    <div style="height: 15rem; width: 15rem" class="img-container">
+      <img
```
Comment (Ankur9669) [inline image/image.html:22]: Here you can add a comment stating the reason that why you have used inline styling, which would help reviewers to understand this

File: input/input.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/input/input.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <h1>Inputs</h1>
+    <div class="input-container">
+      <label for="text-input">Username</label>
+      <input type="text" name="text-input" class="txt-box" required />
+
```
Comment (Ankur9669) [inline input/input.html:15]: Can use id attribute to connect label and input

**Review by Ankur9669 — APPROVED** [review]
Code looks fine to me. Great Work!!!

File: image/image.html [hunk 3]
```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/image/image.css" />
+    <title>Document</title>
```
Comment (zabihshaik) [inline image/image.html:8]: Better to add Title

File: image/image.html [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    <h3>Responsive image</h3>
+    <div class="img-container">
+      <img
+        class="responsive-img"
+        src="https://picsum.photos/700/350"
+        alt="demo-image"
+      />
+    </div>
+    <h3>Round image</h3>
+    <div style="height: 15rem; width: 15rem" class="img-container">
```
Comment (zabihshaik) [inline image/image.html:21]: Better to avoid Inline styles

File: input/input.html [hunk 5]
```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/input/input.css" />
+    <title>Document</title>
```
Comment (zabihshaik) [inline input/input.html:8]: Add title

**Review by zabihshaik — COMMENTED** [review]
Everything looks good. Plz check the comments.

File: image/image.html (hunk 4 above)
Comment (parvez-rashid) [reply image/image.html:21]: Yes, I will remove these in the refactor.

File: input/input.html (hunk 2 above)
Comment (parvez-rashid) [reply input/input.html:15]: Sure, I will do the change.

File: image/image.html (hunk 1 above)
Comment (parvez-rashid) [reply image/image.html:22]: I will actually try to remove this altogether.

**Pull request merged**